book/animations.md

```diff
@@ -1088,15 +1097,15 @@ class Tab:
     def render(self):
         # ...
         for item in self.display_list:
-            print(item.repr_recursive())
+            print_tree(item)
 ```
 
 When run on the [opacity transition
 example](examples/example13-opacity-transition.html) before the animation has
 begun, it should print something like:
 
-    SaveLayer(alpha=0.999): bounds=Rect(13, 18, 787, 40.3438)
```

Comment:
Removing the rect here invalidates the go further block text a little bit; I'l go back and fix it later.

Reply:
Yes—I noticed that but actually the printing the bounds wasn't in the code block cited here! (They're only in your version in the source.) So I agree we should fix up text.